tools/vccexe/vccexe.nim

```diff
@@ -104,8 +104,8 @@ Microsoft (R) C/C++ Optimizing Compiler if no secondary
 command was specified
 """
 
-proc parseVccexeCmdLine(argseq: seq[TaintedString], 
-vccversionArg: var seq[string], printPathArg: var bool, 
+proc parseVccexeCmdLine(argseq: seq[TaintedString],
+vccversionArg: var seq[string], printPathArg: var bool,
```

Comment:
gah, I expected nimpretty to be smarter here. The point is: Use 4 space indentation for breaking up long parameter lists.

Reply:
like as in https://github.com/nim-lang/Nim/pull/13329/commits/956e5ab3a702c6e0127d20f0a238cae271a18541 ?